src/libstore/local-store.cc

```diff
@@ -576,6 +576,17 @@ void LocalStore::checkDerivationOutputs(const StorePath & drvPath, const Derivat
     }
 }
 
+void LocalStore::linkDeriverToPath(const StorePath deriver, const string outputName, const StorePath output)
+{
+    retrySQLite<void>([&]() {
+        auto state(_state.lock());
+        state->stmtAddDerivationOutput.use()
+            (queryValidPathId(*state, deriver))
+            (outputName)
+            (printStorePath(output))
```

Comment:
`output` is an "actual" output here, right? In that case, doesn't this cause confusion wrt the meaning of the `DerivationOutputs` table?